view-user/src/components/common/Button/Button.tsx

```diff
@@ -0,0 +1,28 @@
+import styles from "./Button.module.css";
+import { CgLogOut } from "react-icons/cg";
+import classNames from "classnames";
+
+
+const Button = ({
+  theme = "primary",
+  size = "l",
+  shape = "circle",
+  ...props
+}) => {
+
+  return (
+    <main>
+      <div>
+        <button
+          className={classNames(styles[theme],styles[size],styles[shape])}
+          onClick={() => console.log("button click")}
+        >
+          Logout
+          <CgLogOut className={styles.icon} />
```

Comment:
アイコンと文字を入れ替えてみました。好みかもしれません。
```suggestion
          <span className={styles.contents}>
            <CgLogOut className={styles.icon} />
            <p>Logout</p>
          </span>
```